From cc30986a6033e3b3b8709cf62a84bd832209e493 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Wed, 9 Sep 2020 15:04:14 +0800
Subject: [PATCH] FindOpenCL.cmake: don't look for driver at build time

We don't need the icd driver to be present when building.

Upstream-Status: Inappropriate

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 builder/FindOpenCL.cmake | 2 --
 1 file changed, 2 deletions(-)

diff --git a/builder/FindOpenCL.cmake b/builder/FindOpenCL.cmake
index ed531bc8..d8ac00a6 100644
--- a/builder/FindOpenCL.cmake
+++ b/builder/FindOpenCL.cmake
@@ -25,13 +25,11 @@ set( OCL_LIBS "" )
 
 if ( NOT OPENCL_INCLUDE MATCHES NOTFOUND )
     if ( NOT OPENCL_LIBRARY MATCHES NOTFOUND )
-        if ( EXISTS /etc/OpenCL/vendors/intel.icd )
             set ( OPENCL_FOUND TRUE )
 
             get_filename_component( OPENCL_LIBRARY_PATH ${OPENCL_LIBRARY} PATH )
 
             list( APPEND OPENCL_LIBS OpenCL )
-        endif()
     endif()
 endif()
 
-- 
2.26.2

